.gitignore

```diff
@@ -216,6 +216,7 @@ workflow_data/
 
 # Jupyter Notebooks
 **/.ipynb_checkpoints/
+.virtual_documents/
```

Comment:
can we add a comment on what this directory means, thankks

Reply:
Sure, though I guess we should probably explain what `.ipynb_checkpoints` is as well. Will add now!